### ISSUE TITLE
Feature request: option to rename the default `_prisma_migrations` table

### ISSUE THREAD

**Issue by Papooch** [prisma/prisma#18625]
## Problem

As a cost saving measure (on a dev environment), we want to use a single database for multiple apps (each with their own distinct prisma schema). Since the the migrations table name is fixed, this causes a collision.

## Suggested solution

Add an option like `migrationsTableName` to the `datasource db` section.

```prisma
datasource db {
  provider = "postgresql"
  url = env("DATABASE_URL")
  migrationsTableName = "_my_migrations_table"
}
```

## Alternatives

Using separate logical "databases" within one physical database server, but additional limits and costs apply there depending on the provider. It also only applies to Postgres.

## Additional context

As far as I know, this is possible with both TypeORM and Sequelize migrations.

**Comment by umairb-pedal** [comment]
I second this. Cannot currently run multiple prisma "instances" on the same database. Is it possible to at least not use the "dbo" schema? (For MSSQL).

**Comment by janpio** [comment]
Of course, you can configure the `schema` in the connection string and then the internal migration database is put there: https://www.prisma.io/docs/concepts/database-connectors/sql-server#arguments

**Comment by Papooch** [comment]
I just found out this is a duplicate of an older issue. https://github.com/prisma/prisma/issues/14806. The use-case they describe is basically the same.
